concepts/eventing/performance evaluation/nats-jetstream/results_prod_HA.md

```diff
@@ -0,0 +1,329 @@
+
+```
+Events:
+  Type     Reason                  Age    From                     Message
+  ----     ------                  ----   ----                     -------
+  Normal   Scheduled               8m12s  default-scheduler        Successfully assigned kyma-system/eventing-nats-2 to shoot--kymatunas--fzn-p2-worker-k59z0-z1-5f68f-tnw8q
+  Warning  FailedAttachVolume      8m12s  attachdetach-controller  Multi-Attach error for volume "pv--f20986b4-a4a0-455b-ace2-6b91bdd4f11c" Volume is already exclusively attached to one node and can't be attached to another
+  Warning  FailedMount             6m9s   kubelet                  Unable to attach or mount volumes: unmounted volumes=[eventing-nats-js-pvc], unattached volumes=[eventing-nats-js-pvc config-volume pid kube-api-access-gn9xs]: timed out waiting for the condition
+  Warning  FailedMount             3m52s  kubelet                  Unable to attach or mount volumes: unmounted volumes=[eventing-nats-js-pvc], unattached volumes=[kube-api-access-gn9xs eventing-nats-js-pvc config-volume pid]: timed out waiting for the condition
+  Normal   SuccessfulAttachVolume  113s   attachdetach-controller  AttachVolume.Attach succeeded for volume "pv--f20986b4-a4a0-455b-ace2-6b91bdd4f11c"
+  Warning  FailedMount             97s    kubelet                  Unable to attach or mount volumes: unmounted volumes=[eventing-nats-js-pvc], unattached volumes=[config-volume pid kube-api-access-gn9xs eventing-nats-js-pvc]: timed out waiting for the condition
+```
+
+---
+
+## Test Scenario 2: K8s Cluster Node deleted during test where stream leader NATS pod was deployed
+
+### Run ID: 8/6/2022T14:24 (Duration: 5m, Event Rate: 150rps)
+
+> **NOTE:** Deleted k8s Node where `eventing-nats-2` Pod was deployed after 2 minutes of test execution.
+
+**State before test run:**
+- Stream:
+  - Leader: eventing-nats-2
+  - Replicas: 1
+
+**State after test run:**
+- Stream:
+  - Leader: NA (Stream Unavailable)
+  - Replicas: NA
+
+
+**Test Results:**
+```
+-> Total Events Sent      by **Test Sender** : 10,672 (+ 45 (Failed) = 10,717)
+-> Total Events Received  by **Sink**        : 10,672
+```
+
+**Test Results Dashboard:**
+
+![](assets/ha_crash2_1.png "")
+
+**Test Sender Stats:**
+
+![](assets/ha_crash2_2.png "")
+
+**Finding:** 
+
+Deleting the K8s Node where stream leader NATS Pod was deployed resulted in the JetStream stream to be unavailable as there was only one replica of the stream. NATS stopped accepting any new events for that steam and event-publisher-proxy was failing to publish any event to NATS. The system started to work again as soon as the crashed NATS Pod was deployed on another Node and stream became available again. Therefore, a single stream relpica is not enough to cater a situation of single-node failure.
+
+---
+
+## Test Scenario 3: NATS Pod (JetStream Stream Leader) deleted during test
```

Comment:
You explained the first 2 test cases were nicely in the [Test Cases section](https://github.com/kyma-project/community/blob/8ef445ddf96d4d74f22bafc31e5a439faec814a2/concepts/eventing/performance%20evaluation/nats-jetstream/results_prod_HA.md#test-cases).

It would be interesting now to know which test cases are to be checked now and for what reason 😃

Reply:
After the discovery of downtime with only #replicas and the deleted node being the stream leader I guess you want to do some further research now on this.

Reply:
Ah ok, now you are deleting a pod instead of a node

Reply:
Maybe something like this:

Now that we did test what happens if a k8s node is deleted, we want to see what happens if the NATS pod is deleted.

Reply:
added